_posts/2020-06-13-split-your-rspec-individual-test-examples-between-github-action-nodes.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "Split your RSpec individual test examples between Github Action nodes with Knapsack Pro"
```

Comment:
WIP title - need a better idea

Reply:
The idea for the title with some keywords and keeping Github Actions in the title name closer to the beginning of the title:

* How to run slow RSpec files on Github Actions with parallel jobs by doing an auto split of the spec file by test examples

If we change the title we need to update MD file name for this blog post.